.gitignore

```diff
@@ -1,6 +1,8 @@
 !/.tools/.gitkeep
 !/build/.gitkeep
 !/build/cache/.gitkeep
+/.direnv/
+/.envrc.local
```

Comment:
I have these globally ignored so I do not need to add them in every single project. Consider that you can do that too.